internal/command/meta_config.go

```diff
@@ -163,11 +163,19 @@ func (m *Meta) getInput(ctx context.Context, variable *configs.Variable) (string
 	if !m.Input() {
 		return "", fmt.Errorf("input is disabled")
 	}
+
+	varDesc := variable.Description
+	switch {
+	case variable.Description != "" && variable.DeprecatedSet:
+		varDesc = fmt.Sprintf("%s.\nVariable is marked as deprecated with the following message: %s", variable.Description, variable.Deprecated)
+	case variable.DeprecatedSet:
+		varDesc = fmt.Sprintf("Variable is marked as deprecated with the following message: %s", variable.Deprecated)
+	}
```

Comment:
nit: it seems like a duplicate of the function from `backend_local.go`. We could move it to `configs.Variable` and reuse in both places.